src/Boot/BootstrapManager.php

```diff
@@ -289,7 +289,7 @@ public function doBootstrap($phase, $phase_max = false, AnnotationData $annotati
             if ($phase_index > $bootstrapped_phase) {
                 if ($result = $this->bootstrapValidate($phase_index)) {
                     if (method_exists($bootstrap, $current_phase)) {
-                        $this->logger->log(LogLevel::BOOTSTRAP, 'Drush bootstrap phase: {function}()', ['function' => $current_phase]);
+                        $this->logger->log(LogLevel::INFO, 'Drush bootstrap phase: {function}()', ['function' => $current_phase]);
```

Comment:
Once #4814 lands, it won't be necessary to switch all of these to the standard log levels. You might still want to do this; it would probably be allowable to make a change like this in a minor version, but it might be better to wait until Drush 11.